docs/pages/machine-workload-identity/machine-id/deployment/bound-keypair.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: Deploying Machine ID with Bound Keypair Joining
+description: "How to install and configure Machine ID with Bound Keypair joining"
+---
+
+In this guide, you will install Machine ID's agent, `tbot`, on an arbitrary
+node. This node could be a bare-metal machine, a VM, a container, or any other
```

Comment:
```suggestion
host. This node could be a bare-metal machine, a VM, a container, or any other
```

"node" pretty consistently means "SSH server" in the Teleport docs, so let's avoid that confusion.